RNIronSourceRewardedVideo.js

```diff
@@ -60,10 +60,18 @@ const removeAllListeners = () => {
   IronSourceRewardedVideoEventEmitter.removeAllListeners('ironSourceRewardedVideoAdRewarded');
 };
 
+isRewardedVideoAvailable = async () => {
+  const timeoutTime = 5000;
+  setTimeout(() => Promise.reject(new Error(`Timeout isRewardedVideoAvailable: timeout: ${timeoutTime}`))
+    , timeoutTime);
+  return RNIronSourceRewardedVideo.isRewardedVideoAvailable();
+}
+
 module.exports = {
   ...RNIronSourceRewardedVideo,
   initializeRewardedVideo: () => RNIronSourceRewardedVideo.initializeRewardedVideo(),
   showRewardedVideo: (placementName) => RNIronSourceRewardedVideo.showRewardedVideo(placementName),
+  isRewardedVideoAvailable: () => RNIronSourceRewardedVideo.isRewardedVideoAvailable(callback),
```

Comment:
What is `callback`?

Reply:
Oh myBad I didn't pushed my last modification.